ansible/host_vars/pangu/drone.yml

```diff
@@ -1,10 +1,9 @@
-drone_domain: drone.gitea.io
-drone_orgs: go-gitea
-drone_admins: lafriks,lunny,techknowlogick
+drone_server_host: drone.gitea.io
+drone_user_filter: lafriks,lunny,techknowlogick,go-gitea
```

Comment:
I think it should be just:
```suggestion
drone_user_filter: go-gitea
```

Reply:
There user filter won't be used at all anymore, admins are stored in the database now.

Reply:
As I understand from docs only users/org users listed here will be able to authorize in this instance